data/src/main/java/com/my/kizzy/data/repository/KizzyRepositoryImpl.kt

```diff
@@ -30,14 +29,23 @@ import javax.inject.Inject
 
 class KizzyRepositoryImpl @Inject constructor(
     private val api: ApiService,
+    private val imgurApi: ApiService.ImgurApiService
 ): KizzyRepository {
 
-    override suspend fun getImage(url: String): String? {
-        return api.getImage(url).getOrNull()?.toImageAsset()
+    override suspend fun getImage(url: String, token: String): String? {
```

Comment:
Since we are already accessing Prefs here why not get the token from the prefs too. This would result in so many less changes to the other parts of the app

Reply:
There wouldn't be any issue, thought I saw the Samsung presence and tried to make it consistent